### ISSUE TITLE
Feature Request: Modify options in basket

### ISSUE THREAD

**Issue by Noodleyman** [cubecart/v6#1593]
I have a requirement at the moment to allow customers to modify their option selections within the checkout process. I will do this via custom code but thought it would be a nice addition to the core CubeCart product, thus raising a ticket to see if support can be added for it out of the box in a future release. 

Currently only selected options are listed to review, not modify.

**Comment by abrookbanks** [comment]
Nice! :)

**Comment by briansandall** [comment]
One thing to keep in mind is that options have the potential to alter product weight, price, and possibly other, custom qualities such as product dimensions. Some of those qualities (e.g. weight) would require a re-querying of shipping options if changed.

While I like the idea, I think rather than trying to duplicate all of the logic that occurs during the add-to-cart and checkout processes, it might be more prudent to take the customer back to the product page and have them edit their order that way.

@Noodleyman Of course, if you come up with a viable solution and are willing to share it, that would be awesome!

**Comment by Noodleyman** [comment]
I managed to get it working quite cleanly for my customer, I hooked into the cart update process and captured any submitted options and applied them to the order if different. since this is caught in the  update and before the page reload, the refresh made the various prices recalculate OK.  I used a custom module to make other data available so selected options can be displayed from available options. This was all done in the context of HTML selects since that was the project requirement. It's possible to do.

There isn't much use in me sharing what I came up with as it was bespoke to the customer requirement, however the above concept worked great.
